### DISCUSSION TITLE
How Accurate is the GPS Timing Metadata for Videos?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#592] [General]
This question builds on this discussion:
https://github.com/gopro/labs/discussions/563

I understand from the above cited discussion that the GPS Sync feature gets around +/-50 milliseconds from true time (as best I can put it) when it tries setting the RTC of the HERO11 Black.

Is this the same for the GPS metadata embedded in the video (and presumably the photographs) or is it different?

I am asking since I'm using the [Telemetry Overlay](https://goprotelemetryextractor.com/telemetry-overlay-gps-video-sensors) program which has a gauge for extracting the date and time from the metadata inside video files.

**Comment by dnewman-gpsw** [comment]
With HERO11 it should similar or better accuracy.

**Comment by AlexLandherr** [comment]
Ok. thanks! I've been mostly testing this in a nearby park with little to
obstruct the GPS signal except trees.

Also, as I understand it HERO12 has not GPS receiver.
